test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -8327,7 +8340,7 @@ def make_input(batch_size):
         other_input = make_input(RNN_BATCH_SIZE + 1)
         self.run_test(model, other_input, batch_size=RNN_BATCH_SIZE + 1)
 
-    @disableScriptTest()  # TODO: RuntimeError: Exporting the operator __is_ to ONNX is not supported
+    @skipScriptTest()  # TODO: https://msdata.visualstudio.com/Vienna/_workitems/edit/1253950
```

Comment:
__is_ operator is already added. Is this still failing but for different reason?

Reply:
Yes, currently fails with:
```
Traceback (most recent call last):                                                                                                                                                                                                                                                                                                                                │·········
  File "/home/azureuser/src/pytorch/pytorch/test/onnx/test_pytorch_onnx_onnxruntime.py", line 8360, in test_transformer_encoder                                                                                                                                                                                                                                   │·········
    self.run_test(MyModule(512, 8, 2048 , 0., 3), (x,), atol=1e-6)                                                                                                                                                                                                                                                                                                │·········
  File "/home/azureuser/src/pytorch/pytorch/test/onnx/test_pytorch_onnx_onnxruntime.py", line 347, in run_test                                                                                                                                                                                                                                                    │·········
    _run_test(script_model, scripting_remained_onnx_input_idx, flatten=False)                                                                                                                                                                                                                                                                                     │·········
  File "/home/azureuser/src/pytorch/pytorch/test/onnx/test_pytorch_onnx_onnxruntime.py", line 327, in _run_test                                                                                                                                                                                                                                                   │·········
    return run_model_test(self, m, batch_size=batch_size,                                                                                                                                                                                                                                                                                                         │·········
  File "/home/azureuser/src/pytorch/pytorch/test/onnx/test_pytorch_onnx_onnxruntime.py", line 185, in run_model_test                                                                                                                                                                                                                                              │·········
    ort_sess = convert_to_onnx(model, input=input, opset_version=self.opset_version,                                                                                                                                                                                                                                                                              │·········
  File "/home/azureuser/src/pytorch/pytorch/test/onnx/test_pytorch_onnx_onnxruntime.py", line 88, in convert_to_onnx                                                                                                                                                                                                                                              │·········
    torch.onnx._export(model, input_copy, f,                                                                                                                                                                                                                                                                                                                      │·········
  File "/home/azureuser/src/pytorch/pytorch/torch/onnx/__init__.py", line 50, in _export                                                                                                                                                                                                                                                                          │·········
    result = utils._export(*args, **kwargs)                                                                                                                                                                                                                                                                                                                       │·········
  File "/home/azureuser/src/pytorch/pytorch/torch/onnx/utils.py", line 751, in _export                                                                                                                                                                                                                                                                            │·········
    _model_to_graph(model, args, verbose, input_names,                                                                                                                                                                                                                                                                                                            │·········
  File "/home/azureuser/src/pytorch/pytorch/torch/onnx/utils.py", line 540, in _model_to_graph                                                                                                                                                                                                                                                                    │·········
    graph = _optimize_graph(graph, operator_export_type,                                                                                                                                                                                                                                                                                                          │·········
  File "/home/azureuser/src/pytorch/pytorch/torch/onnx/utils.py", line 229, in _optimize_graph                                                                                                                                                                                                                                                                    │·········
    graph = torch._C._jit_pass_onnx(graph, operator_export_type)                                                                                                                                                                                                                                                                                                  │·········
  File "/home/azureuser/src/pytorch/pytorch/torch/onnx/__init__.py", line 368, in _run_symbolic_function                                                                                                                                                                                                                                                          │·········
    return utils._run_symbolic_function(*args, **kwargs)                                                                                                                                                                                                                                                                                                          │·········
  File "/home/azureuser/src/pytorch/pytorch/torch/onnx/utils.py", line 1104, in _run_symbolic_function                                                                                                                                                                                                                                                            │·········
    return symbolic_fn(g, *inputs, **attrs)                                                                                                                                                                                                                                                                                                                       │·········
  File "/home/azureuser/src/pytorch/pytorch/torch/onnx/symbolic_opset10.py", line 27, in div                                                                                                                                                                                                                                                                      │·········
    return torch.onnx.symbolic_opset9.true_divide(g, self, other)                                                                                                                                                                                                                                                                                                 │·········
  File "/home/azureuser/src/pytorch/pytorch/torch/onnx/symbolic_opset9.py", line 196, in true_divide                                                                                                                                                                                                                                                              │·········
    if sym_help._is_fp(self) or sym_help._is_fp(other):                                                                                                                                                                                                                                                                                                           │·········
  File "/home/azureuser/src/pytorch/pytorch/torch/onnx/symbolic_helper.py", line 340, in _is_fp                                                                                                                                                                                                                                                                   │·········
    type = value.type().scalarType()                                                                                                                                                                                                                                                                                                                              │·········
RuntimeError: rINTERNAL ASSERT FAILED at "../aten/src/ATen/core/jit_type_base.h":540, please report a bug to PyTorch.
```